### ISSUE TITLE
`jupyterlab-execute-time` seems to reset `jupyterlab-vim`/CodeMirror configuration on load

### ISSUE THREAD

**Issue by firai** [deshaw/jupyterlab-execute-time#97]
## Description

When `jupyterlab-execute-time` is installed alongside `jupyterlab-vim`, when a document is loaded, the vim block cursor flashes for a second, but CodeMirror then goes back into normal input mode (without vim) in the first cell. The user must switch to another cell and back in order to enable vim in the first cell. Since the block cursor appears initially (for a second), it seems like `jupyterlab-execute-time` might be resetting the CodeMirror configuration after `jupyterlab-vim` has loaded.

(P.s. I did not experience this issue when I used JL 3 versions of these extensions together.)

## Reproduce

1. Install both `jupyterlab-vim` and `jupyterlab-execute-time`
2. Create a new notebook

## Expected behavior

`jupyterlab-execute-time` does not reset other CodeMirror extensions

## Context

- Extension versions:
jupyterlab-vim 4.0.3
jupyterlab-execute-time 3.0.1
- Operating System and its version: Windows
- Browser and its version: Latest Firefox and Edge

**Comment by krassowski** [comment]
Reproduced. I narrowed it down to:

https://github.com/deshaw/jupyterlab-execute-time/blob/597580e47dd6da2db9392da039e2bad07aa02a0a/src/ExecuteTimeWidget.ts#L54-L61

Manually toggling "Recording timing" in "Notebook" settings also causes this behaviour. Similarly, toggling "Show editor for read-only Markdown cells" or any of the "Code Cell Configuration" options leads to this issue too.

It does not occur when toggling non-editor related options of notebook. It seems that this will need addressing in `jupyterlab-vim` rather than here.
